Users/Glob_Patterns.rst

```diff
@@ -174,14 +209,10 @@ brackets. Parentheses that have no match are ignored as well as
 
```

Comment:
Every '*' is an special character should be preceded by '\' , unless used inside the quotes.' '